runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -183,7 +190,75 @@ benchmarks! {
         );
     }
 
-    delete_channel {
+    channel_update_without_assets {
```

Comment:
I think you should also introduce `let d in 1 .. MAX_BYTES_METADATA;` benchmark parameter here for consistency